app/src/modules/whyis-dataset.js

```diff
@@ -239,7 +239,7 @@ function buildDepictionLd (file, uri) {
     '@id': `${uri}/depiction`,
     '@type': 'http://purl.org/net/provenance/ns#File',
     'http://www.w3.org/2000/01/rdf-schema#label': file.originalname,
-    'http://w3.org/ns/dcat#accessURL': `${window.location.origin}/api/files/${file.filename}?isDirectory=true`
+    'http://w3.org/ns/dcat#accessURL': `${window.location.origin}/api/files/${file.filename}?isFileStore=true`
```

Comment:
I'm getting some strange in-browser errors when I pulled this change and tried it locally, but it may just be my local